features/bootstrap/HydraContext.php

```diff
@@ -192,8 +192,8 @@ public function assertPropertyIsNotReadable(string $propertyName, string $classN
      */
     public function assertPropertyIsWritable(string $propertyName, string $className)
     {
-        if (!$this->getPropertyInfo($propertyName, $className)->{'hydra:writable'}) {
-            throw new ExpectationFailedException(sprintf('Property "%s" of class "%s" is not writable', $propertyName, $className));
+        if (!$this->getPropertyInfo($propertyName, $className)->{'hydra:writeable'}) {
+            throw new ExpectationFailedException(sprintf('Property "%s" of class "%s" is not writeable', $propertyName, $className));
```

Comment:
No need to change this. Both `writable` and `writeable` are correct: https://en.wiktionary.org/wiki/writable

Reply:
>  do I also have to change the text in the behat feature : "property is writable"

No.

Reply:
Okay I removed it